portality/models/background.py

```diff
@@ -84,6 +84,7 @@ def add_audit_message(self, msg, timestamp=None):
             timestamp = dates.now_with_microseconds()
         obj = {"message": msg, "timestamp": timestamp}
         self._add_to_list_with_struct("audit", obj)
+        print(msg)
```

Comment:
Just a reminder note to remove print statements

Reply:
I actually added the print message to make debugging of the background jobs easier. I will add `if app.config.get("DOAJENV") == 'dev'` to make sure it prints only in dev.

Reply:
probably use `if app.config.get("DEBUG") == True` for more generality.